### PR TITLE
Git-Articles completed

### PR DESCRIPTION
**Pull request by DursunER**

### CODE REVIEW

File: __tests__/news.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
+        };
+
+        const articleObj = res.body.article[0];
+
+        expect(articleObj).toEqual(article_3);
```
Comment (douglashellowell) [inline __tests__/news.test.js]: ideally you would not be sending back an `array` when there is only one article

change this line to 

```diff
-   const articleObj = res.body.article[0];
+   const articleObj = res.body.article;
```

File: __tests__/news.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  it("/api/articles/:article_id", () => {
+    return request(app)
+      .get("/api/articles/1000")
+      .expect(404)
+      .then((res) => {
+        const status = res.status;
+        const msg = res.body.err;
+        expect(status).toBe(404);
+        expect(msg).toBe("Article id not found");
+      });
```
Comment (douglashellowell) [inline __tests__/news.test.js]: excellent!

You don't need to be checking the `res.status` as you've already got the `.expect(404)` on `line 37` here

**Comment by douglashellowell** [comment]
Good stuff so far!

You need one more error handling test on this endpoint - what if a user tried to fetch an article using an `article_id` that *wasn't a string*?

i.e `GET /api/articles/not-an-id` 


Remember you can push to this same `Get-Articles` branch and it will automatically update this PullRequest - no need to create a new one!

File: __tests__/news.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  });
+});
```
Comment (emilyb93) [inline __tests__/news.test.js:71]: We need to have some more tests here. Think about what could happen if;
- we request a review id that is not a number ('/api/review/dogs')

**Review by emilyb93 — CHANGES_REQUESTED** [review]
More testing needed

**Comment by douglashellowell** [comment]
ready to merge! 🚀

**Pull request merged**